### PR TITLE
Wait until completion - StartInstance

### PR DESCRIPTION
**Pull request by Podkamienna**
Added waiting until completion of start machine request

### CODE REVIEW

File: pkg/automation/instances.go [hunk 1]
```diff
@@ -55,6 +55,9 @@ func (s *googleService) StopInstance(project string, zone string, instance strin
 // StartInstance starts instance using instances.start method
```
Comment (s17k) [inline pkg/automation/instances.go:55]: It is obvious that StartInstance starts some instance, I would focus on stuff that is hardest to guess from the code.
Like, what permissions are required (very often mentioned in other googleService methods here).

Comment (Podkamienna) [reply pkg/automation/instances.go:55]: The comment is exactly the same as the one in the StopInstance

Comment (s17k) [reply pkg/automation/instances.go:55]: So probably good to change that one too!
https://github.com/googleinterns/recomator/blob/master/pkg/automation/disks.go has nice comments, for example

Comment (Podkamienna) [reply pkg/automation/instances.go:55]: As I was doing some part of the project in the project written mainly by Viktoriia, I decided to follow the convention she chose. The permissions are listed in access.go - and I think that the part of making the code modular is not having to think about them in different places. So I think, that mentioning them would only confuse the person wanting to just check, what is the function for. Moreover, from my experience this kind of comment may also be helpful - especially to someone not familiar with the API.

Comment (s17k) [reply pkg/automation/instances.go:55]: If you want to follow Viktoriia's conventions, the permissions are actually written down above function definitions in many files. But I wouldn't say that following conventions is the end goal.

Comment (Podkamienna) [reply pkg/automation/instances.go:55]: Okay, then I can change all the comments in instances.go to also mention permissions.

**Comment by s17k** [comment]
LGTM, but I don't know the context for this change so probably best if Viktoriia/Artur review it first.

**Review by ViktoriiaKovalova — APPROVED** [review]
LGTM

**Pull request merged**